x-pack/plugins/maps/common/elasticsearch_geo_utils.d.ts

```diff
@@ -13,3 +15,11 @@ export function turfBboxToBounds(turfBbox: unknown): MapExtent;
 export function clampToLatBounds(lat: number): number;
 
 export function clampToLonBounds(lon: number): number;
+
+export function hitsToGeoJson(
+  hits: any[],
```

Comment:
Since flattenHit takes a hit and is typed as `Record<string, unknown>`, then hits should be `Record<string, unknown>[]`.